src/plugins/postal-savings-rs/api.ts

```diff
@@ -0,0 +1,48 @@
+import { accountDetailsToId, convertAccount, convertCardTransactions, convertTransactions } from './converters'
+import { AccountDetails, PBAccount } from './models'
+import { fetchAccountData, fetchCardTransactions } from './fetchApi'
+import { Transaction } from '../../types/zenmoney'
+
+function accountCardKey (accountId: string): string {
+  return `account/${accountId}/card`
```

Comment:
There are two options for the ley format:
1. `account/{accountId}/card`
2. `accountCard/{accountId}`

I've picked the first one because it allows us to bind other data to an account if it is needed